src/box/lua/config/cluster_config.lua

```diff
@@ -209,6 +209,9 @@ local nested_cluster_config = schema.new(schema_name, record_from_fields({
             }),
         }),
     }),
+    include = schema.array({
+        items = schema.scalar({type = 'string'}),
+    }),
```

Comment:
Side note (not a review comment): it would be interesting to have some includes in a group, a replicaset and an instance scope, but it seems that it needs extra work (at least we should understand what to do if the `groups` field is present in a config file that is included from, for example, the replicaset scope).

Reply:
Right now I'm not in a mood to create an issue for this:) also we should create an issue to support cross-source includes (it might be a complex one, as we discussed on call).